frontend/src/DataImporter/CsvUpload.tsx

```diff
@@ -18,9 +21,19 @@ export const CsvUpload = (props: CsvUploadProps) => {
       props.onUpload(content);
     };
     reader.readAsText(file);
+    setFileName(file.name);
   }
 
   return <div>
-    <input type="file" accept=".csv" onChange={handleFileUpload}/>
+    <HiddenFileInput type="file" accept=".csv" id={props.id} onChange={handleFileUpload} />
+    <Button className="mb-3" variant={"outline"} onClick={() => document.getElementById(props.id)!.click()}>
```

Comment:
nie używamy document.getElementById w komponentach Reactowych, to się chyba może rozjechać
`useRef` służy do takich rzeczy

https://react.dev/reference/react/useRef